src/diffUtils.js

```diff
@@ -0,0 +1,4 @@
+export function getModuleDiffUrl(name, from, to) {
+    return `https://diff.intrinsic.com/${name}/${from}/${to}`;
```

Comment:
What if either `from` or `to` has `~` or `^` in it? Will it still work?

Reply:
Seems like versions should be strict:

![image](https://user-images.githubusercontent.com/302213/54188855-ab032e00-44c1-11e9-9d66-345fe2834b60.png)

Reply:
Yeah, it only takes exact versions, due to the permalink nature of a Package Diff URL. Otherwise ~3.0.1 could mean 3.0.2, 3.1.0, etc.

Reply:
I'll get it changed. For some reason I thought that we had exact versions at that point.